migrations/20200415150144-add-school-administration-types-in-profileFields.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  async up(db) {
+    global.migrationMsg = "Add school types and administration types in school and state entity types";
+
+    let schoolTypesForm = {
+      "field" : "schoolTypes",
+      "label" : "School types",
+      "value" : "",
+      "visible" : "",
+      "editable" : "",
+      "input" : "text"
```

Comment:
@aman-tunerlabs - This is multi-select field as it can have multiple values. Also add possible values like primary etc as mentioned in story